HousingManagementSystemApi/Startup.cs

```diff
@@ -77,8 +91,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
         private static void AddHttpClients(IServiceCollection services)
         {
-            AddHttpClient(services, HttpClientNames.HousingSearch, "HOUSING_SEARCH_API_URI", "HOUSING_SEARCH_API_KEY");
             AddHttpClient(services, HttpClientNames.Properties, "PROPERTIES_API_URI", "PROPERTIES_API_KEY");
+            AddHttpClient(services, HttpClientNames.Asset, "HOUSING_ASSET_API_URI", "HOUSING_ASSET_API_KEY");
```

Comment:
Nitpick: Restore original ordering to remove unrelated changes and maintain atomic commit.